guide/04-feature-data-and-analysis/part5_merge_parcels.ipynb

```diff
@@ -84,16 +84,16 @@
   },
```

Comment:
<p>The maps are not showing up in the guide.  Can you republish this guide...but with map cells, you have to do the following</p><p> </p><ul><li>Insert a blank code cell below the map, and run map.take_screenshot().</li><li>replace the map.take_screenshot() code with the code that rendered the map, in this case </li></ul><p><em>map = gis.map("Keowns, WI")</em></p><p><em>map.draw(parcels_subset)</em></p><p><em>map.zoom_to_layer(parcels_subset)</em></p><p><em>map</em></p><ul><li><em>delete the original map cell</em></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>